src/test/java/com/notably/logic/LogicManagerTest.java

```diff
@@ -52,13 +50,11 @@ public void setUp() {
     @Test
     public void execute_invalidCommandFormat_throwsParseException() {
         String invalidCommand = "uicfhmowqewca";
-        assertParseException(invalidCommand, MESSAGE_UNKNOWN_COMMAND);
```

Comment:
Removing this line is similar to just removing the whole test case. The essence of a test case lies in its assertions, which are deleted in this case.

Instead of removing this assertion, let's keep it, but instead supply the message manually

Reply:
Let's refer to this as (3)